openspec/specs/text-operations/spec.md

```
@@ -0,0 +1,101 @@
+# text-operations Specification
+
+## Purpose
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)